hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/fs/http/client/BaseTestHttpFSWith.java

```diff
@@ -2081,6 +2085,33 @@ private void testGetFileLinkStatus() throws Exception {
     assertTrue(fs.getFileLinkStatus(linkToFile).isSymlink());
   }
 
+  private void testGetStatus() throws Exception {
+    if (isLocalFS()) {
+      // do not test the getStatus for local FS.
+      return;
+    }
+    final Path path = new Path("/foo");
+    FileSystem fs = FileSystem.get(path.toUri(), this.getProxiedFSConf());
+    if(fs instanceof DistributedFileSystem){
+      DistributedFileSystem dfs = (DistributedFileSystem)
+          FileSystem.get(path.toUri(), this.getProxiedFSConf());
+      FileSystem httpFs = this.getHttpFSFileSystem();
+
+      FsStatus dfsFsStatus = dfs.getStatus(path);
+      FsStatus httpFsStatus = httpFs.getStatus(path);
+
+      //Validate used free and capacity are the same as DistributedFileSystem
+      Assert.assertEquals(dfsFsStatus.getUsed(), httpFsStatus.getUsed());
+      Assert.assertEquals(dfsFsStatus.getRemaining(),
+          httpFsStatus.getRemaining());
+      Assert.assertEquals(dfsFsStatus.getCapacity(), httpFsStatus.getCapacity());
```

Comment:
There are already static imports for ``assertEquals`` you don't need the ``Assert`` prefix